settings.json

```diff
@@ -0,0 +1,600 @@
+{
+    "type": "object",
+    "properties": {
+        "ALLOWED_HOSTS": {
+            "type": "array",
+            "default": []
+        },
+        "DATABASES": {
+            "type": "object",
+            "properties": {
+                "default": {
+                    "type": "object",
+                    "properties": {
+                        "NAME": {
+                            "type": "string",
+                            "default": "nautobot"
+                        },
+                        "USER": {
+                            "type": "string",
+                            "default": ""
+                        },
+                        "PASSWORD": {
+                            "type": "string",
+                            "default": ""
+                        },
+                        "HOST": {
+                            "type": "string",
+                            "default": "localhost"
+                        },
+                        "PORT": {
+                            "type": "string",
+                            "default": ""
+                        },
+                        "CONN_MAX_AGE": {
+                            "type": "integer",
+                            "default": 300
+                        },
+                        "ENGINE": {
+                            "type": "string",
+                            "default": "django.db.backends.postgresql"
+                        }
+                    }
+                }
+            }
+        },
+        "CACHES": {
```

Comment:
Probably would be good from a maintainability standpoint to alpha-sort the top-level keys?